src/app.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from "@nestjs/common";
+import { CSVtoPDF } from "./tryTable";
+import PDFDocument from "pdfkit-table";
+
+@Injectable()
+export class AppService {
+  getCSVandConvertToPDF(file):PDFDocument {
+    const arr: string[] = file.buffer.toString().split("\n");
+    let arr2d: string[][] = [];
+    let temp: string;
+    let i: number = 0;
+
+    for (temp of arr) {
+      let t: string[] = temp.split(",");
+      arr2d.push([]);
+
+      for (let j = 0; j < t.length; j++) {
+        arr2d[i].push(t[j]);
+      }
+
+      i++;
+    }
+
+    arr2d.pop();
+
+    const createPDF = new CSVtoPDF();
+    
+    return createPDF.conversionTop(arr2d);
+  }
```

Comment:
You may need to put this function into the service file that is new resource based on the comments on `tryTable.ts`